neural_compressor/compression/pruner/criteria.py

```diff
@@ -289,4 +292,49 @@ def on_before_optimizer_step(self):
                 mask_grad = self.modules[key].block_mask.grad.clone()
                 self.collected_grads[key].append(mask_grad)
                 self.scores[key] += mask_grad.pow(2)
+   
+
+@register_criterion('sparse_llm')
+class SparseLLMCriterion(PruningCriterion):
+    """Pruning criterion.
+    
```

Comment:
if this criterion is specially designed for sparsegpt, we'd better move it into sparstgpt class